ci/scripts/flink-bench.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# Exits as soon as any line fails.
+set -euo pipefail
+
+echo "==========================================="
+echo "start the flink bench instance to run the benchmark"
+echo "==========================================="
+aws ec2 start-instances --instance-ids i-029fdf626052dcdaf
```

Comment:
Wouldn't it expose some secret details?

Reply:
No, we need to find if the buildkite workers are in the same vpc as the aws instance, if so then we do not need any pem file or secrets to login.